src/main/java/com/internet/shop/application/Main.java

```diff
@@ -0,0 +1,22 @@
+package com.internet.shop.application;
+
+import com.internet.shop.dao.jdbc.ProductDaoJdbcImpl;
+import com.internet.shop.model.Product;
+
+public class Main {
+    public static void main(String[] args) {
+        ProductDaoJdbcImpl productDaoJdbc = new ProductDaoJdbcImpl();
```

Comment:
- let's use injector)
- we can remove dao annotation above ProductDaoImpl class